packages/embed/README.md

```diff
@@ -26,3 +26,100 @@ To generate SHA384 for a specific asset from an URL, run the following command i
+    oSession["ui-backcolor"] = "Yellow";
+}
+```
+
+### Testing on Azure Portal
+
+Until we move everyone to the new embed page, your bot might not using the new embed page on Azure Portal.
+
+Append the following code in FiddlerScript `OnPeekAtResponseHeaders` function. It will force your bot to use Gemini regardless of the deployment phase.
+
+When combined with the FiddlerScript above, "Test in Web Chat" in Azure Portal will point to the locally-developed version of the Gemini page.
+
+```js
+if (oSession.uriContains("https://webchat.botframework.com/embed/your-bot-id?features=webchatpreview&t=")) {
+    oSession["ui-backcolor"] = "Yellow";
+    oSession.oResponse.headers.HTTPResponseCode = 302;
+    oSession.oResponse.headers.HTTPResponseStatus = '302 Moved';
+    oSession.oResponse.headers.Add('location', oSession.PathAndQuery.replace('your-bot-id?', 'your-bot-id/gemini?b=your-bot-id&v=4.3&'));
+}
+```
+
+## FAQs
+
+These are common questions around the new Gemini page.
+
+### What is the version of Web Chat I am using on my page?
+
+Press <kbd>F12</kbd> and look at the console. It should say something similar to the following:
+
+```
+Web Chat: Selecting version "default" -> "4" -> "4.3" -> "4.3.0".
+```
+
+The line above means, the version specifier is not specified. And Web Chat use "default" version, which forward to major version 4, then minor version 4.3, then patch version 4.3.0.
+
+### Why I am not seeing Web Chat v4 on my page?
+
+Look at the <kbd>F12</kbd> console log and see how version selection is being done.
+
+If you don't see the line, it is probably because the new Gemini page is not rolled out completely. If your bot is in, the URL should become https://webchat.botframework.com/embed/your-bot-id/gemini. If your bot is not in, it will not have `/gemini` appended.
+
+### I want to go back to previous versions
+
+There are a handful of versions we support:
+
+- Majors
+   - `4` will serve the latest minor version of `4.*`. As of time of this writing, it is `4.3.0`
+   - `3` will serve the latest minor version of `0.*`. As of time of this writing, it is `0.15.1-v3.748a85f`
+   - `scorpio` will serve the version before Gemini page is introduced. As of time of this writing, it is `0.11.4-ibiza.f373b1d`
+      - This will be same as "Enable preview" checkbox checked
+   - `aries` will serve the version before Gemini page is introduced
+      - This will be same as "Enable preview" checkbox unchecked
+      - This version will show a bot icon and welcome message on the top of the page
+- Minors
+   - `4.2` will serve the latest patch version of `4.2.*`. As of time of this writing, it is `4.2.0`
+
+However, we do not support specifying patch versions, e.g. `4.2.1`. This is because patch versions are for important security fixes, we will roll it out to users who are on the same minor version and guarantee security fixes will not introduce breaking changes.
```

Comment:
```suggestion
However, we do not support specifying patch versions, e.g. `4.2.1`. This is because patch versions are for important security fixes. We will roll it out to users who are on the same minor version and we guarantee that security fixes will not introduce breaking changes.
```